src/content/indices/monaco.mdx

```diff
@@ -0,0 +1,76 @@
+---
+name: Monaco
+indexName: Indice Qualité de l'air
+indexAcronym: IQA
+---
+
+import ColorScale from "@components/ColorScale.astro";
+import BreakpointsTable from "@components/BreakpointsTable.astro";
+import Admonition from "@components/Admonition.astro";
+
+
+## Background
+
+Monaco (Principality of Monaco) assesses and communicates urban air quality through their national Indice Qualité de l'air (IQA), developed in collaboration 
+with AtmoSud, the regional air monitoring agency. The AQI ranges from 0 (very good) to 100 (alert threshold) and is based on real-time data collected from 
+five monitoring stations across the Principality. In addition to current conditions, forecasted AQI values for the following day are also provided, integrating 
+meteorological predictions. 
+
+Monaco’s IQA methodology is designed to be straightforward, which reflects public exposure to key urban pollutants. It considers three major pollutants: 
+nitrogen dioxide (NO<sub>2</sub>), fine particulate matter (PM<sub>10</sub>), and ozone (O<sub>3</sub>). 
+
+## Color scale
+
+The IQA uses 10 color-coded index levels that fall under six broader air quality categories. Each color represents a specific range of AQI values and is 
+designed to provide a quick visual cue of pollution severity and associated health risks. 
```

Comment:
Instead of include the french category named in the csv, lets list them out here with english translation in parentheses e.g. bon (Good)